### PR TITLE
docs: Update Introduction

### PR DESCRIPTION
**Pull request by rayjh**
Update Introduction section of Intro-setup.rst

### CODE REVIEW

File: docs/Intro-Setup.rst [hunk 1]
```diff
@@ -3,83 +3,89 @@
...
 
-OpenPOWER Cluster Genesis (OPCG) enables greatly simplified configuration of clusters of
+Cluster POWER-Up enables greatly simplified configuration of clusters of
 bare metal OpenPOWER servers running Linux. It leverages widely used open
 source tools such as Cobbler, Ansible and Python. Because it relies
 solely on industry standard protocols such as IPMI and PXE boot, hybrid
 clusters of OpenPOWER and x86 nodes can readily be supported. Currently
-OPCG supports Ethernet networking with separate data and management
-networks. OPCG can configure simple flat networks for typical HPC
+Cluster POWER-Up supports Ethernet networking.  Cluster POWER-Up can
```
Comment (jaywcarman) [inline docs/Intro-Setup.rst]: Single space new sentences.

File: docs/Intro-Setup.rst [hunk 2]
```diff
@@ -3,83 +3,89 @@
...
+configure simple flat networks for typical HPC
 environments or more advanced networks with VLANS and bridges for
-OpenStack environments. OPCG also configures the switches in the
-cluster. Currently Mellanox SX1410 is supported for the data network and
-the Lenovo G8052 is supported for the management network.
+OpenStack environments. Complex Heterogenious clusters can be easily deployed
```
Comment (jaywcarman) [inline docs/Intro-Setup.rst]: ~~Heterogenious~~ heterogenous

File: docs/Intro-Setup.rst [hunk 3]
```diff
@@ -3,83 +3,89 @@
...
-renamed to ease installation of additional software stack elements.
-
-Management and/or Data switches can be set to "passive" mode to allow
-deployment without supplying login credentials to the switch management
-interfaces. This mode requires the user to manually write switch MAC address
-tables to file and to configure the management and/or data switch in accordance
-with the defined networks. The client networks will still be configured by
-Cluster Genesis.
+Cluster POWER-Up supports Cisco, Mellanox and Lenovo switches. Not all
+functionailty is enabled on all switch types.  Currently redundant
```
Comment (jaywcarman) [inline docs/Intro-Setup.rst]: ~~functionailty~~ functionality

File: docs/Intro-Setup.rst [hunk 4]
```diff
@@ -3,83 +3,89 @@
...
+networking (MLAG) is only implemented on Mellanox switches. Port channel
+support is only implemented on Cisco (NX-OS) and Mellanox switches.
+POWER-Up can configure any number of node interfaces on cluster nodes.
+To facilitate installation of higher level software, interfaces can be
+optionally renamed.
+
+Interface templates are used to define network configurations
+in the config.yml file.  These can be physical ports, bonded ports,
+Linux bridges or vLANS.  Interface templates can be entered using
```
Comment (jaywcarman) [inline docs/Intro-Setup.rst]: ~~vLANS~~ VLANS

File: docs/Intro-Setup.rst [hunk 5]
```diff
@@ -3,83 +3,89 @@
...
+Ubuntu or Red Hat network configuration syntax.  Once defined, interface
+templates can be applied to any node template.  Node interfaces can
+optionally be configured with static ip addresses.  These can be assigned
```
Comment (jaywcarman) [inline docs/Intro-Setup.rst]: ~~ip~~ IP

File: docs/Intro-Setup.rst [hunk 6]
```diff
@@ -3,83 +3,89 @@
...
-file (config.yml) which drives the cluster configuration. The
-configuration file is a yaml text file which the user edits. Example
-YAML files are included. The configuration process is driven from a
-"deployer" node which does not need to remain in the cluster when
-finished. The process is as follows;
+Cluster POWER-Up is designed to be easy to use. If you are implementing
+one of the supported architectures with supported hardware, it eliminates
+the need for custom scripts or programming. It does this via a text
+configuration file (config.yml) which drives the cluster configuration.
+The configuration file is a yaml text file which the user edits. Several
```
Comment (jaywcarman) [inline docs/Intro-Setup.rst]: ~~yaml~~ YAML (uppercase acronyms?)

File: docs/Intro-Setup.rst [hunk 7]
```diff
@@ -3,83 +3,89 @@
...
+Example config files are included docs directory. The configuration
```
Comment (jaywcarman) [inline docs/Intro-Setup.rst]: ~~Example~~ example

File: docs/Intro-Setup.rst [hunk 8]
```diff
@@ -3,83 +3,89 @@
...
+The POWER-Up software supports clusters of servers
 interconnected with Ethernet. The
-servers must support IPMI and PXE boot. Currently single racks with single
-or redundant data switches (with MLAG) are supported. Multiple racks can
-be interconnected with traditional two tier access-aggregation
-networking.  In the future we plan to support two tier leaf-spine networks
-with L3 interconnect capable of supporting VXLAN.
+servers must support IPMI and PXE boot. Multiple racks can
+be configured with traditional two tier access-aggregation
+networking.  POWER-Up configures both a management and
```
Comment (jaywcarman) [inline docs/Intro-Setup.rst]: single space new sentences (lines 55, 58, 65, 73, 74, 75, 76, 77, 86 also)

File: docs/Intro-Setup.rst (hunk 1 above)
Comment (rayjh) [reply docs/Intro-Setup.rst]: fixed

File: docs/Intro-Setup.rst (hunk 2 above)
Comment (rayjh) [reply docs/Intro-Setup.rst]: fixed

File: docs/Intro-Setup.rst (hunk 3 above)
Comment (rayjh) [reply docs/Intro-Setup.rst]: fixed

File: docs/Intro-Setup.rst (hunk 5 above)
Comment (rayjh) [reply docs/Intro-Setup.rst]: fixed

File: docs/Intro-Setup.rst (hunk 6 above)
Comment (rayjh) [reply docs/Intro-Setup.rst]: fixed

File: docs/Intro-Setup.rst (hunk 7 above)
Comment (rayjh) [reply docs/Intro-Setup.rst]: fixed

File: docs/Intro-Setup.rst [hunk 9]
```diff
@@ -92,7 +92,7 @@ dedicated to specific purposes.
 Supported Hardware
 ~~~~~~~~~~~~~~~~~~~
 
-**Compute Nodes**
+:**Compute Nodes**
```
Comment (jaywcarman) [inline docs/Intro-Setup.rst]: looks like ":" got added my mistake

Comment (rayjh) [reply docs/Intro-Setup.rst]: gremlins

**Pull request merged**